salt/states/junos.py

```diff
@@ -188,29 +186,28 @@ def rollback(name, id, **kwargs):
 
     """
     ret = {"name": name, "changes": {}, "result": True, "comment": ""}
-    ret["changes"] = __salt__["junos.rollback"](id, **kwargs)
+    ret["changes"] = __salt__["junos.rollback"](id=id, **kwargs)
     return ret
 
 
 @resultdecorator
-def diff(name, d_id):
+def diff(name, d_id, **kwargs):
```

Comment:
Since adding the kwargs parameter to the interface, should add 
.. versionchanged:: Sodium

see https://docs.saltstack.com/en/latest/topics/development/conventions/documentation.html

Reply:
Thanks @dmurphy18 Changed via https://github.com/saltstack/salt/pull/55824/commits/9bc4f8c4244533119671202bcbe27e7bda495d0d#diff-442286d51776ddefc791bc7f15e42fc2R196